app/src/main/res/layout/activity_gallery.xml

```diff
@@ -3,10 +3,19 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:background="@drawable/background"
     tools:context=".SettingsActivity">
 
     <!-- Toolbar -->
     <include layout="@layout/toolbar"/>
 
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/gallery_recyclerview"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
Change layout height to display all the images at the end of the gallery.
I don't know why, but it seems like Top to bottom of the toolbar is not enough to make gallery start after toolbar, thus I added paddingTop with the height of the toolbar
```suggestion
        android:layout_height="match_parent"
        android:paddingTop="80dp"
```